.nvmrc

```diff
@@ -1 +1 @@
-6
+8
```

Comment:
upgrading our minimum node version so that we can use more modern JS constructs in the build script and the tests